mlflow/genai/scorers/phoenix/utils.py

```diff
@@ -0,0 +1,86 @@
+"""Utility functions for Phoenix integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from mlflow.entities.trace import Trace
+from mlflow.genai.utils.trace_utils import (
+    extract_retrieval_context_from_trace,
+    parse_inputs_to_str,
+    parse_outputs_to_str,
+    resolve_expectations_from_trace,
+    resolve_inputs_from_trace,
+    resolve_outputs_from_trace,
+)
+
+
+def map_scorer_inputs_to_phoenix_record(
+    metric_name: str,
+    inputs: Any = None,
+    outputs: Any = None,
+    expectations: dict[str, Any] | None = None,
+    trace: Trace | None = None,
+) -> dict[str, Any]:
+    """
+    Convert MLflow scorer inputs to Phoenix evaluator record format.
+
+    Args:
+        metric_name: Name of the Phoenix metric
+        inputs: The input to evaluate
+        outputs: The output to evaluate
+        expectations: Expected values and context for evaluation
+        trace: MLflow trace for evaluation
+
+    Returns:
+        Dictionary formatted for Phoenix evaluator
+    """
+    if trace:
+        inputs = resolve_inputs_from_trace(inputs, trace)
+        outputs = resolve_outputs_from_trace(outputs, trace)
+        expectations = resolve_expectations_from_trace(expectations, trace)
+
+    record = {}
+
+    # Parse inputs to string
```

Comment:
nit: let's remove these one-liners as they don't make the code that much more readable